pyintegration/tests/test_uri.py

```diff
@@ -0,0 +1,33 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+
+import unittest
+
+import jpy
+
+from deephaven2._jcompat import j_hashset
+from deephaven2.table import Table
+from deephaven2.uri import resolve
+from tests.testbase import BaseTestCase
+
+
+class UriTestCase(BaseTestCase):
+    def test_uri_local_table(self):
```

Comment:
Do we have a way to test a URI that returns a JPY unwrapped object?

Reply:
Actually looked into this when writing the test cases. There didn't seem to be a way to initialize other URI resolvers than the one for CSV.  Getting a case to work will take significant amount of work and not worth it ATM.